src/pages/home/Home.page.tsx

```diff
@@ -49,6 +49,13 @@ const statData: StatData[] = [
     icon: HiOutlineMail,
     percentage: "30%",
```

Comment:
Tip - The `icon` property is currently using `HiOutlineMail` for multiple items. If each stat item is supposed to have a unique icon, make sure to update this icon to something that accurately represents the data it's linked with. Otherwise, consider if reusing the same icon is intentional for design consistency.